src/main/java/com/iridium/iridiumskyblock/listeners/ItemCraftListener.java

```diff
@@ -37,4 +39,13 @@ public void monitorItemCraft(CraftItemEvent event) {
         island.ifPresent(value -> IridiumSkyblock.getInstance().getIslandManager().incrementMission(value, "CRAFT:" + material.name(), amount));
     }
 
+    @EventHandler(priority = EventPriority.LOW)
+    public void noUseCrystalCraft(PrepareItemCraftEvent event) {
```

Comment:
More better name :panda:
```suggestion
    public void onItemCraft(PrepareItemCraftEvent event) {
```